### DISCUSSION TITLE
watch_file flake.nix and watch_file flake.lock in the .envrc

### DISCUSSION THREAD

**Discussion by Horiodino** [direnv/direnv#1285] [Q&A]
what is the purpose of `watchfile`  ? and how its different than `use flakes`

**Comment by Horiodino** [comment]
like : ```watch_file flake.nix  
watch_file flake.lock ```

**Reply by zimbatm** [reply]
See https://direnv.net/man/direnv-stdlib.1.html#codewatchfile-ltpathgt-ltpathgt-code

use flake watches the flake.nix by default, by also loads the flake
